Sriram/Reg.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <title>Regno</title>
+  </head>
+  <body>
+    <h2><strong>Register</strong></h2>
+
+    <form>
+      First name:
+      <input type="text" name="firstname" value="First name" />
+      Last name:
+      <input type="text" name="lastname" value="Last name" /><br /><br />
+      Gender:
+      <label for="male">Male</label>
+      <input type="radio" name="gender" value="male" checked /> Male
+      <label for="female">Female</label>
+      <input type="radio" name="gender" value="female" /> Female
+      <label for="other">Other</label>
+      <input type="radio" name="gender" value="other" /> Other<br /><br />
+      Email:
+      <input type="text" name="email" required />
+      Password:
+      <input type="password" name="psw" />
+      Birthday: <input type="date" name="bday" /> <input type="submit" /><br />
+     I agree to terms and conditions <input type="checkbox" name="vehicle" ><br>
+
+      <br /><br />
```

Comment:
Why is the name "vehicle"?

Reply:
Yeah I removed that now

Reply:
> Yeah I removed that now

You should always have a name. Else the form data won't be transmitted.